packages/gamut-labs/src/experimental/Popover/index.tsx

```diff
@@ -0,0 +1,156 @@
+import cx from 'classnames';
+import React, { useState, useCallback, useEffect, useRef } from 'react';
+import { useWindowSize, useClickAway, useWindowScroll } from 'react-use';
+import { useHotkeys } from 'react-hotkeys-hook';
+
+import styles from './styles.module.scss';
+import { BodyPortal } from '../../../../gamut/src/BodyPortal';
+
+export type PopoverProps = {
+  children: React.ReactElement<any>;
+  className?: string;
+  /**
+   * Whether the popover is rendered.
+   */
+  isOpen: boolean;
+  /**
+   * The target element around which the popover will be positioned.
+   */
+  targetRef: React.RefObject<
+    Pick<HTMLDivElement, 'getBoundingClientRect' | 'contains'>
+  >;
+  /**
+   * Which vertical edge of the source component to align against.
+   * @default left
+   */
+  align?: 'left' | 'right';
+  /**
+   * Number of pixels to offset the popover from the source component.
+   * @default 20
```

Comment:
Nit: you should be able to remove these; storybook should (now or soon) deduce them from the `=` defaults below. They tend to get out of date so I think it's good not to explicitly rewrite them.

```suggestion
   * @default 20
```

Reply:
it does show up in storybook from the `=` defaults now, yay!